src/contexts/TodosList.js

```diff
@@ -0,0 +1,53 @@
+import { createContext, useState } from "react";
+export const TodosListContext = createContext({});
+
+export default function TodosListContextProvider({ children }) {
+  const [data, setData] = useState([]);
```

Comment:
I would suggest naming the context and the file to `TodosContext` instead of `TodosListContext`